.github/workflows/ci.yml

```diff
@@ -38,9 +38,44 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [16]
-        os: [ubuntu-latest, windows-latest]
-        e2e-browser: ['chromium']
+        include:
+          - node-version: 16
+            os: [ubuntu-latest, windows-latest]
+            e2e-browser: 'chromium'
+          - node-version: 18
```

Comment:
now that we have this we can delete it from cross-browser tests on line 86 as we're currently running those tests twice